core/src/main/java/io/questdb/cutlass/json/AbstractJsonParser.java

```diff
@@ -0,0 +1,107 @@
+package io.questdb.cutlass.json;
+
+import io.questdb.std.MemoryTag;
+import io.questdb.std.Misc;
+import io.questdb.std.Mutable;
+import io.questdb.std.ObjectPool;
+import io.questdb.std.QuietCloseable;
+import io.questdb.std.Unsafe;
+import io.questdb.std.Vect;
+import io.questdb.std.str.AbstractCharSequence;
+import io.questdb.std.str.DirectUtf8Sink;
+
+public abstract class AbstractJsonParser implements JsonParser, Mutable, QuietCloseable {
```

Comment:
why do we have abstract class? Preferences parser is the only descendant

Reply:
ent uses this class too

Reply:
a comment has been added to this class that it should not be removed